frontend/src/cljs/imas_seamap/db.cljs

```diff
@@ -91,21 +86,38 @@
-                     :habitat-observations-url (str api-url-base "habitat/habitatobservations")
-                     :layer-previews-url    (str media-url-base "layer_previews/")
-                     :story-maps-url        (str wordpress-url-base "wp-json/wp/v2/story_map?acf_format=standard")}})
+   :config          {:url-paths {:layer                 "layers/"
+                                 :base-layer            "baselayers/"
+                                 :base-layer-group      "baselayergroups/"
+                                 :organisation          "organisations/"
+                                 :classification        "classifications/"
+                                 :region-stats          "habitat/regions/"
+                                 :descriptor            "descriptors/"
+                                 :save-state            "savestates"
+                                 :category              "categories/"
+                                 :keyed-layers          "keyedlayers/"
+                                 :amp-boundaries        "habitat/ampboundaries"
+                                 :imcra-boundaries      "habitat/imcraboundaries"
+                                 :meow-boundaries       "habitat/meowboundaries"
+                                 :habitat-statistics    "habitat/habitatstatistics"
+                                 :bathymetry-statistics "habitat/bathymetrystatistics"
+                                 :habitat-observations  "habitat/habitatobservations"
+                                 :layer-previews        "layer_previews/"
+                                 :story-maps            "wp-json/wp/v2/story_map?acf_format=standard"}
+                     :url-bases {"http://localhost:3451/"
+                                 {:api-url-base       "http://localhost:8000/api/"
+                                  :media-url-base     "http://localhost:8000/media/"
+                                  :wordpress-url-base "http://localhost:8888/"
+                                  :img-url-base       "/img/"}
+                                 "https://seamapaustralia.org/map/"
+                                 {:api-url-base       "https://data.imas.utas.edu.au/seamap/api/"
+                                  :media-url-base     "https://data.imas.utas.edu.au/seamap/media/"
+                                  :wordpress-url-base "https://seamapaustralia.org/"
+                                  :img-url-base       "/app/img/"}
+                                 "https://seamapaustralia-dev.imas.utas.edu.au/map"
+                                 {:api-url-base       "https://data-dev.imas.utas.edu.au/seamap/api/"
+                                  :media-url-base     "https://data-dev.imas.utas.edu.au/seamap/media/"
+                                  :wordpress-url-base "https://seamapaustralia-dev.imas.utas.edu.au/"
+                                  :img-url-base       "/app/img/"}}
```

Comment:
(discussed offline, but leaving a paper-trail here).  This is nicely data-driven, but it still leaves the problem that for a new deployment we have to update the source.  I'd like these kind of external concerns to be configuration variables; no need for a new build, or a source change -- just update the external configuration (which in our case is probably the html page / javascript variables)